src/main/java/wolox/training/repositories/UsersRepository.java

```diff
@@ -0,0 +1,20 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Users;
+
+@Repository
+public interface UsersRepository extends CrudRepository<Users, Long> {
+
+	public Optional<Users> findByName(String name);
+
+	public Iterable<Users> findAll();
+
+	public Optional<Users> findById(Long id);
+
+	public Users save(Users user);
+
+	public void delete(Users user);
+}
```

Comment:
This methods are not necessary. Using CrudRepository you have some methods automatically, like `save, findById, delete, findAll` and others. Che this link https://www.baeldung.com/spring-data-repositories